exe-unit/components/gsb-http-proxy/src/http_to_gsb.rs

```diff
@@ -74,44 +83,52 @@ impl HttpToGsbProxy {
         };
 
         let msg = GsbHttpCallMessage {
-            method,
-            path,
+            method: method.clone(),
+            path: path.clone(),
             body,
             headers: Headers::default().filter(&headers),
         };
 
-        let response = match &self.binding {
-            BindingMode::Local => bus::service(&self.bus_addr).call(msg).await,
-            BindingMode::Net(binding) => {
-                ya_net::from(binding.from)
-                    .to(binding.to)
-                    .service(&self.bus_addr)
-                    .call(msg)
-                    .await
-            }
-        };
+        let endpoint = self.endpoint();
 
-        let result = response.unwrap_or_else(|e| Err(HttpProxyStatusError::from(e)));
+        log::info!("Proxy http {method} `{path}` call to [{}]", endpoint.addr());
```

Comment:
micronit: wouldn't it be cleaner to implement `fmt::Display` for `GsbHttpCallMessage`?

Reply:
I added this in last commit. It is not so elegant, because this Display doesn't print all fields but it's ok

Reply:
I always thought that `Debug` is for all fields :smiley:

Reply:
I'm not using this Display impl everywhere, since I don't want to clone body